assembl/lib/sqla.py

```diff
@@ -1690,12 +1691,14 @@ def orm_update_listener(mapper, connection, target):
         return
     session = object_session(target)
     if session.is_modified(target, include_collections=False):
+        reindex_content(target, 'update')
```

Comment:
Should we really (re)index an object on every creation/update/deletion? If the indexation code fails, does it make the database operation fail? If yes, isn't it a problem? I would say that on indexation failure, the operation should still work, only the indexation of this object should be retried later. Would it be a good idea to debounce reindexation, in order to avoid congestion problems? Or does elasticsearch handles this by itself? A type of example case I'm thinking about is some very frequent user action in the UI that would update an object in the database each time: for this kind of actions we don't want to trigger (re)indexation as often as the action. These actions could include things like user likes a message (cf https://github.com/assembl/assembl/pull/79/commits/41f5582b3f8c7539747ab88fa904b3e0c9e128d4 ), user marks a message as read (even more when we will add a feature which automatically marks messages as read when they are displayed for a certain amount of time on screen.